opacus/data_loader.py

```diff
@@ -87,12 +87,8 @@ def __init__(
     def from_data_loader(
         cls, data_loader: DataLoader, distributed: bool = False, generator=None
     ):
-        if isinstance(data_loader, cls):
-            # TODO: this should be exception, not assert
-            assert data_loader.distributed == distributed
```

Comment:
Why was this removed?

Reply:
What's the upside of having it? Looks like we can easily re-create DPDataLoader from another DPDataLoader and change distributed flag, so why not do it